pkg/store/github/github.go

```diff
@@ -103,11 +104,15 @@ func (s *GitHubStore) ReloadCache(ctx context.Context) error {
 			return err
 		}
 
-		versions := make([]*core.ModuleVersion, len(tags))
-		for i, tag := range tags {
-			versions[i] = &core.ModuleVersion{
-				Version:   strings.TrimPrefix(tag.GetName(), "v"), // Terraform uses SemVer names without 'v' prefix
-				SourceURL: fmt.Sprintf("git::ssh://git@github.com/%s/%s.git?ref=%s", owner, name, tag.GetName()),
+		versions := make([]*core.ModuleVersion, 0)
+		for _, tag := range tags {
+			ver := strings.TrimPrefix(tag.GetName(), "v") // Terraform uses SemVer names without 'v' prefix
+			_, err := version.NewSemver(ver)
+			if err == nil {
+				var element = new(core.ModuleVersion)
+				element.Version = ver
+				element.SourceURL = fmt.Sprintf("git::ssh://git@github.com/%s/%s.git?ref=%s", owner, name, tag.GetName())
+				versions = append(versions, element)
```

Comment:
Om du synes det ser penere ut kan man også
```go
versions = append(versions, &ModuleVersion{
    Version:   ver,
    SourceURL: fmt.Sprintf("git::ssh://git@github.com/%s/%s.git?ref=%s", owner, name, tag.GetName())
})
```
Så blir det mer kompakt, og en variabel mindre å finne et bra navn til.

Reply:
Tried that and didn't get it to work, but seems to work now